### PR TITLE
[DEVOPS-1009] Import iohk-ops docs from private wiki

### PR DESCRIPTION
**Pull request by rvl**
These are good docs for any user of iohk-ops and they needn't be locked away in a private wiki.

### CODE REVIEW

File: docs/Daedalus-Installer-Update-Procedure.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
+# Daedalus Installer Update Procedure
+
+After the installers are built in CI, they need to be copied to the
+installer updates S3 bucket, and an update proposal needs to be
+submitted to the cardano-sl network.
+
+This is how wallet clients will self-update.
+
+**New!** There have recently been some changes and improvements to
```
Comment (angerman) [inline docs/Daedalus-Installer-Update-Procedure.md]: Should this have a date? When is *new* not *new* anymore?

File: docs/Daedalus-Installer-Update-Procedure.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+these commands from [DEVOPS-868][].
+
+## Running the scripts
```
Comment (angerman) [inline docs/Daedalus-Installer-Update-Procedure.md]: Should there be a prerequisits section here? What do I need to actually perform these steps? And if I don't have the prerequisits whom should I contact to get them? SSH key auth, ...

File: docs/Daedalus-Installer-Update-Procedure.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+    daedalusRevision: 32daff63e1fb1590cd7320e4253e61b2a47b0963
+    lastKnownBlockVersion: '0.1.0'
+
+
+## 2. Download installer files from CI
+
+    io -c NETWORK.yaml update-proposal find-installers DATE
+
+The *DATE* parameter should be the same as what was used for the
+`init` step.
```
Comment (angerman) [inline docs/Daedalus-Installer-Update-Procedure.md]: What if `DATE` was not provided but inferred? Does does it infer the current date here as well?

File: docs/Daedalus-Installer-Update-Procedure.md [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+
+5. The installer filenames should look normal, have the right network
+   and versions, and correspond to the same build as was approved for
+   release by QA (if updating mainnet installers).
+
+These values will also be summarised in the file `wiki.md` within the
+work dir.
+
+**Important**: If an update proposal is made with the wrong
+`applicationVersion`, the update mechanism will be useless until each
```
Comment (angerman) [inline docs/Daedalus-Installer-Update-Procedure.md]: useless means will not work?

File: docs/Daedalus-Installer-Update-Procedure.md [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+## 6. Propose the update and vote in favor using majority stake
+
+Find the IP address of a *privileged relay* with `io info`. This is
+normally private info so don't leak it.
+
+    io -c NETWORK.yaml update-proposal submit DATE --relay-ip 1.2.3.4 [--with-linux]
+
+By default, installers will be proposed for Windows and macOS, but not
+Linux. Use the `--with-linux` flag to include these installers in the
+update proposal.
```
Comment (angerman) [inline docs/Daedalus-Installer-Update-Procedure.md]: Do we want to note, why `--with-linux` is not the default?

File: docs/Daedalus-Installer-Update-Procedure.md [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+
+This will generate a new node db, copy keys from the top-level `keys`
+directory, then "rearrange" the copied keys.
+
+It will then send a transaction to the given relay.
+
+Note the proposal ID which is printed at the end of the output.
+
+If the update proposal was successful, the ratified proposal will take effect in 12 hours. See [Alex Vieth's](https://input-output-rnd.slack.com/archives/C2VJ41WDP/p1522951431000512) analysis of the code to corroborate the expected 12 hour delay.
```
Comment (angerman) [inline docs/Daedalus-Installer-Update-Procedure.md]: Is the slack.com link stable or should we inline the commen into a markdown file alongside thie repository as well?

File: docs/iohk-ops-reference.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+- `static/id_buildfarm`
+- `static/github_token`
+
+### Lack of memory
+
+If you don't have enough memory to build, then set the `iohk-ops --initial-heap-size` option (value is in gibibytes), and trim
+[`topology-production.yaml`](https://github.com/input-output-hk/iohk-ops/blob/master/topology-production.yaml).
+
+To trim the topology file:
+1. delete all entries except `c-a-1`, `r-a-1`, `p-b-1`.
```
Comment (angerman) [inline docs/iohk-ops-reference.md]: A note what those X-Y-N values are would be useful? 
> delete all entries except ..., which are ...?

**Comment by rvl** [comment]
@angerman Thanks for the valuable review comments. I have addressed each one by amending the text.

**Pull request merged**